### DISCUSSION TITLE
希望能把脚注的格式改成没有圈的数字

### DISCUSSION THREAD

**Discussion by YangLeiSX** [sjtug/SJTUThesis#685] [提问] [answered]
使用本科生毕业论文模板。
脚注是带有圈的数字，但是实际使用中感觉不太好看。
特别是附近有参考文献的方括号数字的时候，看起来觉得非常的奇怪。
希望能把脚注的格式改成没有圈的。
（我自己在cls文件中找了一圈没改出来

**Comment by skyzh** [comment, marked as answer]
参考 https://www.overleaf.com/learn/latex/Footnotes, 可以使用 ` \renewcommand{\thefootnote}{\arabic{footnote}}` 把脚注改成喜欢的样式。
